app/_hub/bitnami/microsoft_azure/index.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
@beltran-rubo I've moved it to a new file - Please provide the required metadata and feel free to align the wording now that it's a dedicated page, if you want.

Feel free to just comment if you're too busy and I'll make the changes for you!

Reply:
Thanks! Feel free to populate & merge the pending information. I can do a full revision after that.

Reply:
I'm going on holiday as of tomorrow actually so feel free to give all relevant information so I can add it afterwards. Most of the metadata I'm not sure what to fill in since I don't work for Bitnami..